crates/api_models/src/admin.rs

```diff
@@ -163,33 +163,101 @@ pub struct MerchantConnectorId {
     pub merchant_id: String,
     pub merchant_connector_id: i32,
 }
-//Merchant Connector Account CRUD
-#[derive(Debug, Clone, Serialize, Deserialize)]
+
+/// Create a new Payment Connector for the merchant account. The connector could be a payment processor / facilitator / acquirer or specialized services like Fraud / Accounting etc."
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct PaymentConnectorCreate {
+    /// Type of the Connector for the financial use case. Could range from Payments to Accounting to Banking.
+    #[schema(example = "payment_processor,payment_vas")]
     pub connector_type: api_enums::ConnectorType,
+    /// Name of the Connector
+    #[schema(value_type = String,example = "stripe,adyen")]
     pub connector_name: String,
```

Comment:
1. The examples here would make it seem that the field accepts a comma-separated list of values, which is not the case. Please include only one value as an example for these fields.
2. Explicitly specifying `value_type` for `String` is not required.